packages/curve-ui-kit/src/themes/design/0_primitives.ts

```diff
@@ -101,7 +101,7 @@ export const Sizing = {
 
 export const Duration = {
   Snackbar: 6000,
-  Tooltip: { Enter: 500, Exit: 500 },
+  Tooltip: { Enter: 500, Exit: 50000 },
```

Comment:
Oops forgot this one
```suggestion
  Tooltip: { Enter: 500, Exit: 500 },
```